### ISSUE TITLE
Keyboard shortcuts to check/reveal don't work on Mac?

### ISSUE THREAD

**Issue by mjbaldwin** [downforacross/downforacross.com#307]
Hi, I'm trying to use the advertised keyboard shortcuts:

Alt + S, W, or P	Check Square, Word, or Puzzle
Alt + Shift + S, W, or P	Reveal Square, Word, or Puzzle

But I can't get them to work on my Mac -- not on Chrome and not on Safari.

Obviously Macs don't have an Alt key, they have an Option key, but they're generally the same. But I tried each of Cmd, Opt, and Ctrl with S, W, P, and none of them do a check or reveal.

Am I missing something on how to get this working? Or is it a bug?

Thanks!

**Comment by mjbaldwin** [comment]
Yup. In case it helps, since I'm assuming you're not on Macs -- I used [this link](https://js-keyevents-demo.vercel.app/) to generate a series of keyboard events for Opt+S on my Mac in Chrome:

<img width="1261" alt="image" src="https://github.com/downforacross/downforacross.com/assets/160959/92470fbd-7c84-4258-a3c9-bb27b640b0b2">

And for Opt+Shift+S:

<img width="1255" alt="image" src="https://github.com/downforacross/downforacross.com/assets/160959/580ee3a8-51f2-45d7-9bc0-7b575252d599">

I think your approach will work fine.

(The alternative approach would be to use KeyDown instead of KeyPress, where event.keyCode will still return the ASCII key, and you check the modifiers. But that would probably involve more extensive code modification.)

**Comment by EkanshdeepGupta** [comment]
Bumping this issue. Would be a huge QoL improvement!

**Comment by EkanshdeepGupta** [comment]
Since the fix already exists in @kumar-ish's branch: https://github.com/kumar-ish/downforacross.com/tree/madhav/mac-handle-alt, I'm wondering what can be done to merge it in.
